app/lib/actions/middleware/watch_delegated_proxy_sub_tasks.rb

```diff
@@ -46,7 +46,7 @@ def process_task_results(tasks, results)
         notify ::Actions::ProxyAction::ProxyActionMissing.new, missing if missing.any?
 
         stopped = present.select { |task| %w[stopped paused].include? results.dig(task.remote_task_id, 'state') }
-        notify ::Actions::ProxyAction::ProxyActionStopped.new, stopped if stopped.any?
```

Comment:
Do I understand right, that this should correspond with https://github.com/theforeman/foreman-tasks/blob/124e9c64296c4ed997eabfdc2ad6c2384d4c0d6a/app/lib/actions/proxy_action.rb#L48-L69 ?

Reply:
Yes, exactly